css/main.css

```diff
@@ -0,0 +1,230 @@
+}
+
+/* HEADER */
+
+.top {
+    background-color: #5c7b99;
+    height: 120px;
+    background-size: cover;
+    background-position: center;
+}
+
+nav {
+    float: right;
+    padding: 55px 25px 0 25px;
+}
+
+nav ul li {
+    display: inline-block;
+}
+
+nav ul li a {
+    text-decoration: none;
+    font-family: 'montserrat';
+    font-size: 24px;
+    padding-left: 25px;
+    color: white;
+}
+
+.cover {
+    height:740px;
+    background: url('../img/heading-pic.jpg');
+    background-size: cover;
+    background-position: center;
+}
+
+.cover h1 {
+    margin: 0 auto;
+    text-align: center;
+    font-size: 55px;
+    font-family: lora;
+    padding: 300px 0 0 350px;
+}
+
+.cover h2 {
+    font-size: 36px;
+    text-align: center;
+    font-family: 'montserrat';
+    font-weight: normal;
+    padding: 0 200px 0 450px;
+}
+
+/* Portfolio Section */
+#portfolio {
+    height: 1150px;
+    background-size: cover;
+}
+
+.project {
+    background-color: #ebb9ad;
+    height: 80px;
+    background-size: cover;
+    margin: 0 auto;
+}
+
+.project h3 {
+    font-size: 36px;
+    text-align: center;
+    font-family: lora;
+    color: white;
+    margin: 0 auto;
+    padding: 17px 0 17px 0;
+}
+
+.box {
+    height: 350px;
+    width: 600px;
+    float: left;
+}
+
+.screenshot {
+    height: 300px;
+    Width: 490px;
+    padding: 15px 50px 15px 50px;
+}
+
+.proj-title h4 {
+    font-size: 28px;
+    font-family: lora;
+    margin: 50px 0 0 0;
+}
+
+#portfolio p {
+    padding: 10px 0 10px 0;
+    margin: 0;
+}
+
+.description {
+    text-align: justify;
+    padding-right: 50px;
+}
+
+/* ABOUT ME */
+
+#about {
+    background-color: #ebb9ad;
+    height: 650px;
+    background-size: cover;
+}
+
+#about h3 {
+    font-size: 36px;
+    text-align: center;
+    font-family: lora;
+    color: white;
+    margin: 0 auto;
+    padding: 25px 0 25px 0;
+}
+
+#about img {
+    Width: 50%;
+    padding-left: 150px;
+}
+
+#about p {
+    font-size: 18px;
+    font-family: montserrat;
+    font-weight: normal;
+    color: white;
+    padding: 10px 60px 10px 60px;
+    text-align: justify;
+}
+
+#about h4 {
+    font-size: 24px;
+    font-family: montserrat;
+    font-weight: bold;
+    color: white;
+    text-align: center;
+    text-decoration: underline;
+}
+
+#about ul {
+    padding-left: 150px;
+}
+
+#about ul li {
+    text-decoration: none;
+    font-family: 'montserrat';
+    font-size: 18px;
+    padding-left: 5px;
+    color: white;
+    line-height: 40px;
+
+}
+
+/* CONTACT */
+#contact {
+    height: 350px;
+    background-color: #5c7b99;
+}
+
+#contact h3 {
+    font-size: 36px;
+    font-family: lora;
+    color: white;
+    text-align: center;
+    margin: 0 0;
+    padding: 25px 0 25px 0;
+}
+
+#contact h5 {
+    font-size: 20px;
+    font-weight: normal;
+    color: white;
+    text-align: center;
+}
+
+#contact p {
+    font-size: 18px;
+    font-weight: normal;
+    color: white;
+    text-align: center;
+}
+
+#contact a {
```

Comment:
If you want to make your links have effects on hover and click and what not, this is a great resource for that:

https://www.w3schools.com/css/css_link.asp

For good UX, it's ideal if your links:
1) Have a different, distinguishing style/color from normal text on the website (though this isn't super necessary when what you have there is an email, more important for other types of links/clickable things)
2) React to the users cursor (outside of the boring hand pointer cursor)

So you can just have it change color on a:hover or something!